devel/revive-hf.lua

```diff
@@ -0,0 +1,40 @@
+-- Revives the specified historical figure
+
+--[====[
+
+devel/revive-hf
+=============
+
+Revivs the specified historical figure, even if off-site.
```

Comment:
In the docs for this script (which should be in a .rst file in the docs dir), could you include *why* you'd want to revive a historical figure? Give readers some idea about what situations this would be useful.